tgui/packages/tgui/interfaces/OperatingComputer.js

```diff
@@ -0,0 +1,534 @@
+
+import { useBackend, useLocalState } from '../backend';
+import { Box, ColorBox, Chart, Section, Stack, Tabs, Table } from '../components';
+import { Window } from '../layouts';
+import { HealthStat } from './common/HealthStat';
+import { COLORS } from '../constants';
+import { ReagentGraph } from './common/ReagentInfo';
+import { getStatsMax, processStatsData } from './EngineStats';
+import { capitalize, spaceUnderscores } from './common/stringUtils';
+
+export const OperatingComputer = (props, context) => {
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 1);
```

Comment:
tab view should be synched for all viewers; it's only displaying one screen at a time